docs/source/tutorials/llama_kd_tutorial.rst

```diff
@@ -0,0 +1,228 @@
+.. _llama_kd_label:
+
+============================
+Distilling Llama3 8B into 1B
+============================
+
+This guide will teach you about knowledge distillation (KD) and show you how you can use torchtune to distill a Llama3.1 8B model into Llama3.2 1B.
+If you already know what knowledge distillation is and want to get straight to running your own distillation in torchtune,
+you can jump to knowledge distillation recipe in torchtune, `knowledge_distillation_single_device.py <https://github.com/pytorch/torchtune/blob/main/recipes/knowledge_distillation_single_device.py>`_.
```

Comment:
nit
```suggestion
you can jump to the knowledge distillation recipe in torchtune, `knowledge_distillation_single_device.py <https://github.com/pytorch/torchtune/blob/main/recipes/knowledge_distillation_single_device.py>`_.
```

Reply:
Actually, it might be better to just reference the latter parts of this tutorial for people who want to jump ahead rather than pointing to the recipe file,

Reply:
Changed to link the tutorial section instead of recipe.